scripts/brofish-ping.sh

```diff
@@ -107,7 +106,7 @@ for ((retry=0; retry<$TOTAL_RETRIES; retry++)); do
 done
 
 $ping_ok || {
-  /home/pi/firewalla/scripts/firelog -t cloud -m "brofish ping failed(LOG:$result_log, CPU:$result_cpu, RSS:$result_rss), restart brofish now"
+  /home/pi/firewalla/scripts/firelog -t cloud -m "brofish ping failed(HB:$result_log, CPU:$result_cpu, RSS:$result_rss), restart brofish now"
```

Comment:
Changes look good. Just a minor comment on function/variable names for consistency.
Since we change the checking logic from "log" to "heartbeat", would it be more consistent to do the following renames?
*  brofish_log() => brofish_gb()
*  result_log => result_hb